rust/src/model/sensors/sensor.rs

```diff
@@ -0,0 +1,35 @@
+use rand::Rng;
+use scylla::{FromRow, ValueList};
+use serde::Serialize;
+use uuid::Uuid;
+
+use crate::model::pet::Pet;
+use crate::model::sensors::sensor_type::SensorType;
+
+#[derive(Clone, ValueList, FromRow, Serialize)]
+pub struct Sensor {
+    pub pet_id: Uuid,
+    pub sensor_id: Uuid,
+    pub r#type: SensorType,
+}
```

Comment:
Please use `SerializeRow` instead of deprecated `ValueList`.

Reply:
Also, soon (in the next Rust driver's release) `FromRow` is going to be deprecated and replaced with `DeserializeRow`.